ERROR - 2022-05-06 00:09:16 --> Error connecting to the database: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
CRITICAL - 2022-05-06 00:09:16 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
#0 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 D:\Program Files\xampp\htdocs\mainEChannel\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 D:\Program Files\xampp\htdocs\mainEChannel\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 D:\Program Files\xampp\htdocs\mainEChannel\app\Controllers\Login.php(26): CodeIgniter\BaseModel->first()
#5 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(832): App\Controllers\Login->create()
#6 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\Program Files\xampp\htdocs\mainEChannel\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-05-06 00:20:18 --> Error connecting to the database: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
CRITICAL - 2022-05-06 00:20:18 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
#0 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 D:\Program Files\xampp\htdocs\mainEChannel\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 D:\Program Files\xampp\htdocs\mainEChannel\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 D:\Program Files\xampp\htdocs\mainEChannel\app\Controllers\Login.php(26): CodeIgniter\BaseModel->first()
#5 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(832): App\Controllers\Login->create()
#6 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\Program Files\xampp\htdocs\mainEChannel\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-05-06 03:29:19 --> Duplicate entry 'limod@mailinator.com' for key 'email'
CRITICAL - 2022-05-06 03:29:19 --> Duplicate entry 'limod@mailinator.com' for key 'email'
#0 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Us...', 0)
#1 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Us...')
#2 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Us...')
#3 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Us...', Array, false)
#4 D:\Program Files\xampp\htdocs\mainEChannel\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Program Files\xampp\htdocs\mainEChannel\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\Program Files\xampp\htdocs\mainEChannel\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\Program Files\xampp\htdocs\mainEChannel\app\Controllers\Register.php(59): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(832): App\Controllers\Register->create()
#9 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\Program Files\xampp\htdocs\mainEChannel\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-05-06 03:42:11 --> Duplicate entry 'practitioner5@gmail.com' for key 'email'
CRITICAL - 2022-05-06 03:42:11 --> Duplicate entry 'practitioner5@gmail.com' for key 'email'
#0 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Us...', 0)
#1 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Us...')
#2 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Us...')
#3 D:\Program Files\xampp\htdocs\mainEChannel\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Us...', Array, false)
#4 D:\Program Files\xampp\htdocs\mainEChannel\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Program Files\xampp\htdocs\mainEChannel\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\Program Files\xampp\htdocs\mainEChannel\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\Program Files\xampp\htdocs\mainEChannel\app\Controllers\Register.php(59): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(832): App\Controllers\Register->create()
#9 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 D:\Program Files\xampp\htdocs\mainEChannel\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\Program Files\xampp\htdocs\mainEChannel\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
